source/includes/_embed2.md.erb

```diff
@@ -451,6 +465,8 @@ window.adaEmbed.start({
 
 Used to initialize Embed2 on your page. This action is triggered by default internally, so you will likely not need to call it directly unless you are using Embed2 is `lazy` mode, or have called `stop` and want to re-start Embed2.
 
+`StartOptions` matches anything in `Embed2 API Reference` (e.g. `adaReadyCallback`).
```

Comment:
`StartOptions` isn't a very good name since you can also set these on `window.adaSettings`. Hopefully, this line helps in the meantime.